core/bourbon/helpers/_lightness.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
+
+@function _gamma($channel) {
+  @if $channel < 0.03928 {
+    @return $channel / 12.92;
+  } @else {
+    $c: ($channel + 0.055) / 1.055;
+    // very similar to pow($c, 2.4)
```

Comment:
This comment could become part of SassDoc documentation comments, if useful. We tend to avoid inline Sass comments.